src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,113 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String createWithNameAndCountryRequest =
+                "INSERT INTO manufacturers (name, country) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createManufacturerNameCountryStatement =
+                        connection.prepareStatement(createWithNameAndCountryRequest,
+                             Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerNameCountryStatement.setString(1, manufacturer.getName());
+            createManufacturerNameCountryStatement.setString(2, manufacturer.getCountry());
+            createManufacturerNameCountryStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturerNameCountryStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String getRequest = "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllStatement =
+                        connection.prepareStatement(getRequest)) {
+            getAllStatement.setLong(1, id);
+            ResultSet resultSet = getAllStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getManufacturerFromResultSet(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new RuntimeException("Can not get data from DB ", e);
```

Comment:
```suggestion
            throw new RuntimeException("Can not get manufacturer from DB by id: " + id, e);
```

we need to make exception messages more informative